src/screens/Learn/Faq/Section.tsx

```diff
@@ -13,6 +22,13 @@ const Section = (props: { section: FaqSection }) => {
           <Fragment key={`faq-question-${i}`}>
             <ItemName id={question.questionId}>{question.question}</ItemName>
             <MarkdownContent source={question.answer} />
+            <SmallShareButtons
+              shareUrl={url + question.questionId}
```

Comment:
it's better to use `urlJoin` here instead of concatenating

Reply:
echoing this ^

Reply:
👍